From a1432cbc1bed49c4f33d669b2b8834bdfeadbad5 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@ycbcr.xyz>
Date: Mon, 30 Mar 2020 08:33:30 +0200
Subject: [PATCH 36/36] tools: update meson to 0.54.0

And remove the patch that was merged upstream.

(cherry picked from commit 50ac74cc8572831875ee387fe07668ebdf410940)
---
 extras/tools/SHA512SUMS   | 2 +-
 extras/tools/packages.mak | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/extras/tools/SHA512SUMS b/extras/tools/SHA512SUMS
index f5799daad4..0db2eab2ee 100644
--- a/extras/tools/SHA512SUMS
+++ b/extras/tools/SHA512SUMS
@@ -15,5 +15,5 @@ e80ace766e145f6486e76da1a5a9819221b7f406745a02529b4ad220ef7f51ddd67f23d0d8b187bf
 bbdc23e7772e49da1c7c47e66d4e4efbfbfe9b21dbc59bf3ad9a6e573eecac6c9f52c7f11a64be9897e8deb99ef7ba015164aa8232aa391b901dd7db03632412  bison-3.0.4.tar.xz
 e9785f3d620a204b7d20222888917dc065c2036cae28667065bf7862dfa1b25235095a12fd04efdbd09bfd17d3452e6b9ef953a8c1137862ff671c97132a082e  flex-2.6.4.tar.gz
 8d23dde18525dccaa648ca01df40151e7f00cec4846bd611c8970dbcfc1fb57a453facfe4d41462e7c3c8bb548d44b961a04e4fc3073ab6b65063e53f42bf6fd  nasm-2.14.tar.gz
-0c96c354bcd7e6945473c7df0ddff929ef2bae9c2dfc7b48c6c6174c7f2be4f798398929f9c4d5986aa5fc882305ff76371ad0d65a499f058b33b05ff7025859  meson-0.53.1.tar.gz
+e3c97ffd9409a543e45c8be7b12d4e8437de8dbd0cd236fbe092952d7d0833728d46ff6b679c8a73dae4c4016fdf38b43b56f3959a95968a29db109ebfe254e7  meson-0.54.0.tar.gz
 1650bf9e3eddeb0b0fbb415c2b8e0a7c094421e991fa8139fd77fae0f6ee7ee980b7cf5e98d883c3a884f99abcb06fa26e3980af3a3a5bb6dd655124755782c2  ninja-1.8.2.tar.gz
diff --git a/extras/tools/packages.mak b/extras/tools/packages.mak
index 0e058475c4..d8e533778d 100644
--- a/extras/tools/packages.mak
+++ b/extras/tools/packages.mak
@@ -54,7 +54,7 @@ BISON_URL=$(GNU)/bison/bison-$(BISON_VERSION).tar.xz
 FLEX_VERSION=2.6.4
 FLEX_URL=https://github.com/westes/flex/releases/download/v$(FLEX_VERSION)/flex-$(FLEX_VERSION).tar.gz
 
-MESON_VERSION=0.53.1
+MESON_VERSION=0.54.0
 MESON_URL=https://github.com/mesonbuild/meson/releases/download/$(MESON_VERSION)/meson-$(MESON_VERSION).tar.gz
 
 NINJA_VERSION=1.8.2
-- 
2.26.0.windows.1

